instrumentation/metric/registry_test.go

```diff
@@ -12,12 +12,12 @@ import (
 )
 
 func TestInMemoryRegistry_ExportAll(t *testing.T) {
-	registry := NewRegistry()
-	gauge := registry.NewGauge("hello")
-	gauge.Add(1)
-
-	gaugeValue := registry.ExportAll()["hello"].(gaugeExport)
-	require.EqualValues(t, gaugeValue.Value, 1)
+	//registry := NewRegistry()
```

Comment:
Probably a good idea to uncomment this

Reply:
i mean to rewrite export all ... this test is very vague (create one guage export it and check that there is one)

Reply:
i am going to refactor export so the test will be different.